README.md

```diff
@@ -44,15 +100,87 @@ and configure your IDE accordingly. For VS-CODE, the following can be added to .
       ]
     }
   ]
+  ```
 
 These instructions are for a local docker environment.
 
-### Running Tests
+### Running unit tests
+
 In order to run tests locally you will need to do the following:
+
 1. Navigate to /registered-email-address-web/
```

Comment:
They're using git submodules??

Reply:
Lots of our repos use this submodule: https://github.com/search?p=1&q=submodule+api-enumerations+org%3Acompanieshouse&type=Code

Reply:
Yep indeedy - `api-enumerations`

Reply:
I'd forgotten about this. Still not sure that git submodules are a good idea usually, but I lack context.